server/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -24,6 +24,7 @@
 import org.elasticsearch.common.bytes.ReleasableBytesReference;
 import org.elasticsearch.common.collect.ImmutableOpenMap;
 import org.elasticsearch.common.geo.GeoPoint;
+import org.elasticsearch.common.io.stream.Writeable.Writer;
```

Comment:
nit: is this import strictly necessary? the existing qualified Writeable.Writer is more symmetric with Readable.Writer

Reply:
good catch, will revert 👍